test/run_pass/test_require_main_foo.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
This test is not included in the test sets :(. 

As a suggestion, the file could be put in fixtures folder and then being required to cover the false case.

Reply:
This only be invoked by `test_require_main.js`, but I have not found the `fixtures` folder, will I create one?

Reply:
Samples could be found: `test/run_pass/require1`, `test/run_pass/require2`, `test/run_pass/require3`

Reply:
Yes, it's concise and clear, I will do it.